lldb/packages/Python/lldbsuite/test/lldbtest.py

```diff
@@ -2254,21 +2251,18 @@ def complete_from_to(self, str_input, patterns, turn_off_re_match=False):
                 for idx in range(1, num_matches + 1):
                     compare_string += match_strings.GetStringAtIndex(idx) + "\n"
 
+        # If the singular pattern is the same as the input, assume that we
```

Comment:
This is describing how the function handles its input, maybe it would be better to put it up with the triple-quote doc comment at the beginning of the function?

Reply:
Yes, done.